docs/variants/novacustom_nv4x/post_install.md

```diff
@@ -26,26 +26,43 @@ enable the touchpad hotkey to work under Linux, follow the steps below:
 After executing these steps, it should be possible to enable and disable the
 touchpad using the touchpad hotkey (Fn+F1) on the keyboard when using GNOME.
 
+## Nvidia drivers (Ubuntu 22.04)
+
+For proper functioning of the sleep mode on Ubuntu 22.04, it is required to
+install additional Nvidia drivers.
+
+1. Install drivers by executing the following command on Ubuntu 22.04:
```

Comment:
Maybe the better option is to write, where the command should be executed (in the terminal), than on which system *that info we have in the header).